commons-numbers-core/src/test/java/org/apache/commons/numbers/core/ArithmeticUtilsTest.java

```diff
@@ -435,6 +435,25 @@ void testIsPowerOfTwo() {
         }
     }
 
+    @Test
+    void testPowOptimize() {
```

Comment:
This should be moved above the `testIsPowerOfTwo` method. I would rename it to `testPowEdgeCases`.

Reply:
@aherbert done.